docs/cloud-native-security/index.rst

```diff
@@ -126,76 +122,48 @@ include: review of the customer’s runbook, summary of findings and
-customer. Rackspace will only respond to findings that are at or above the
-defined minimum severity level. Findings with lower severity level will not
-be processed by Rackspace and the customer won’t be charged for those. The
-minimum severity level will be agreed upon between the customer and Rackspace.
-
-Initial selection of the minimum severity level for Rackspace response can
-either be based on a review of the native security products in the
-customer’s environment (if they are already enabled) or simply be set to the
-highest severity level (i.e. Critical/High) to start with. As part of the
-regular monthly account reviews, the minimum severity level will be reviewed
-by Rackspace and the customer. When the current setting does not generate too
-many findings, we can reduce the minimum severity level and start responding
-to findings with a lower severity. Likewise, if the current setting is
-generating too many findings, we can increase the minimum severity level.
-
-From time to time, customers might want Rackspace to investigate a finding
-that is below the defined minimum severity level. Such functionality is
-available directly in the Security Hub user interface of the AWS console
-on the master account. Select a finding and choose
-**Send to Rackspace** in the **Actions** drop-down menu, the finding is
-sent to the Rackspace SIEM and investigated by a specialist. Standard
-charges apply for findings sent to Rackspace using this method. In
-order to protect customers from unintended over-charges, the
-**Send to Rackspace** action supports single findings only. In order to
-send multiple findings to Rackspace, customers should repeat the
-process for each finding individually.
+Findings and Events
+-------------------
+
+Native security products produce findings (rather than raw data points). Every security finding aggregated by Security Hub includes an associated normalized severity level – Critical, High, Medium, Low, Informational.
+
+Findings that are at or above the defined minimum severity level are sent to Rackspace SIEM, where they go through correlation rules and transformed into events. Every event includes one or more correlated findings. Rackspace security specialists respond to these events.
+
+From time to time, customers may wish Rackspace to investigate a finding that is below the defined minimum severity level. Such functionality is available directly in the Security Hub user interface of the AWS console on the master account. Selecting a finding and choosing **Send to Rackspace** in the **Actions** drop-down menu will send the finding to the Rackspace SIEM to be investigated by a specialist. Standard charges apply for findings sent to Rackspace using this method for customers who opted for the per-event billing model. Please note: The **Send to Rackspace** action supports single findings only. In order to send multiple findings to Rackspace, customers should repeat the process for each finding individually.
```

Comment:
... customers might want Rackspace to investigate ...
... drop-down menu sends the finding ...
... billing model. **Note:** The **Send to Rackspace** action ...
... only. To send multiple  ...

Reply:
updated